lib/config.rb

```diff
@@ -57,12 +57,14 @@ def self.setting_files(config_root, env)
     [
       File.join(config_root, 'settings.yml').to_s,
       File.join(config_root, 'settings', "#{env}.yml").to_s,
-      File.join(config_root, 'environments', "#{env}.yml").to_s,
-
-      File.join(config_root, 'settings.local.yml').to_s,
-      File.join(config_root, 'settings', "#{env}.local.yml").to_s,
-      File.join(config_root, 'environments', "#{env}.local.yml").to_s
-    ].freeze
```

Comment:
Can you make sure that we still end up with a frozen array? I'm not sure if it is a big deal or not, but it might bite somebody if we stop freezing it.

Reply:
Woah, didn't mean to remove that. Good catch, fixed.